packages/bff/src/routers/authorizationRouter.ts

```diff
@@ -62,29 +61,27 @@ const authorizationRouter = (
         const err = makeApiProblem(
           error,
           sessionTokenErrorMapper,
-          logger,
+          ctx.logger,
           "Error creating a session token"
         );
 
         return res.status(err.status).send();
       }
     })
     .post("/support", async (req, res) => {
-      const { correlationId, logger } = fromAppContext(req.ctx);
+      const ctx = fromBffAppContext(req.ctx, req.headers);
 
       try {
         const saml = Buffer.from(req.body.SAMLResponse, "base64").toString();
-        const jwt = await authorizationService.samlLoginCallback(
-          correlationId,
-          saml
-        );
+
+        const jwt = await authorizationService.samlLoginCallback(saml, ctx);
         return res.redirect(
           302,
           `${config.samlCallbackUrl}#saml2=${req.body.SAMLResponse}&jwt=${jwt}`
         );
       } catch (error) {
-        logger.error(`Error calling support SAML - ${error}`);
-        makeApiProblem(error, emptyErrorMapper, logger);
+        ctx.logger.error(`Error calling support SAML - ${error}`);
```

Comment:
You can move this error as the last param of the `makeApiProblem` as in the function above